swap_meet/vendor.py

```diff
@@ -1,2 +1,91 @@
+from .item import Item
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item 
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item 
+        else:
+            return False
+    
+    def get_by_category(self, category):
+        return [item for item in self.inventory if item.category == Item(category=category).category]
+    
+    def swap_items(self, Vendor, my_item, their_item):
```

Comment:
be careful using a class name here instead of a lowercase parameter. By using `Vendor.remove`, it suggests that the class is calling the method, which means all Vendor instances might be affected, not the instance itself.